views/js/runner/plugins/security/fullScreen.js

```diff
@@ -310,6 +339,9 @@ define([
                 testRunner
                     .on('exit', function() {
                         doc.removeEventListener(fullScreenEventName, handleFullScreenChange);
+                        if(isIE11()) {
+                            window.removeEventListener('resize', handleFullScreenChange);
```

Comment:
**nitpick:** Caution, the `resize` event is a heavy duty one. It may screw up the browser. Please consider adding a throttling.

**suggestion:** 
```
const throttledHandleFullScreenChange = _.debounce(handleFullScreenChange, delay);
```

**thought:** The event listener should be unregistered at some point. Please consider adding a uninstall process.